Index: es-app/src/EmulationStation.h
===================================================================
--- es-app/src/EmulationStation.h.orig
+++ es-app/src/EmulationStation.h
@@ -7,7 +7,7 @@
 #define PROGRAM_VERSION_MAINTENANCE  1
 #define PROGRAM_VERSION_STRING "2.0.1a"
 
-#define PROGRAM_BUILT_STRING __DATE__ " - " __TIME__
+#define PROGRAM_BUILT_STRING "Relased March 2015"
 
 #define RESOURCE_VERSION_STRING "2,0,1\0"
 #define RESOURCE_VERSION PROGRAM_VERSION_MAJOR,PROGRAM_VERSION_MINOR,PROGRAM_VERSION_MAINTENANCE
