diff -up groovymame-0.226/3rdparty/genie/build/gmake.linux/genie.make.systemlua groovymame-0.226/3rdparty/genie/build/gmake.linux/genie.make
--- groovymame-0.226/3rdparty/genie/build/gmake.linux/genie.make.systemlua	2019-10-30 18:38:08.390000188 +0100
+++ groovymame-0.226/3rdparty/genie/build/gmake.linux/genie.make	2019-10-30 18:44:01.284852520 +0100
@@ -59,45 +59,12 @@ ifeq ($(config),release)
   LIBDEPS            +=
   LDDEPS             +=
   LDRESP              =
-  LIBS               += $(LDDEPS) -ldl -lm
+  LIBS               += $(LDDEPS) -ldl -lm -llua
   EXTERNAL_LIBS      +=
   LINKOBJS            = $(OBJECTS)
   LINKCMD             = $(CC) -o $(TARGET) $(LINKOBJS) $(RESOURCES) $(ARCH) $(ALL_LDFLAGS) $(LIBS)
   OBJRESP             =
   OBJECTS := \
-	$(OBJDIR)/src/host/lua-5.3.0/src/lapi.o \
-	$(OBJDIR)/src/host/lua-5.3.0/src/lauxlib.o \
-	$(OBJDIR)/src/host/lua-5.3.0/src/lbaselib.o \
-	$(OBJDIR)/src/host/lua-5.3.0/src/lbitlib.o \
-	$(OBJDIR)/src/host/lua-5.3.0/src/lcode.o \
-	$(OBJDIR)/src/host/lua-5.3.0/src/lcorolib.o \
-	$(OBJDIR)/src/host/lua-5.3.0/src/lctype.o \
-	$(OBJDIR)/src/host/lua-5.3.0/src/ldblib.o \
-	$(OBJDIR)/src/host/lua-5.3.0/src/ldebug.o \
-	$(OBJDIR)/src/host/lua-5.3.0/src/ldo.o \
-	$(OBJDIR)/src/host/lua-5.3.0/src/ldump.o \
-	$(OBJDIR)/src/host/lua-5.3.0/src/lfunc.o \
-	$(OBJDIR)/src/host/lua-5.3.0/src/lgc.o \
-	$(OBJDIR)/src/host/lua-5.3.0/src/linit.o \
-	$(OBJDIR)/src/host/lua-5.3.0/src/liolib.o \
-	$(OBJDIR)/src/host/lua-5.3.0/src/llex.o \
-	$(OBJDIR)/src/host/lua-5.3.0/src/lmathlib.o \
-	$(OBJDIR)/src/host/lua-5.3.0/src/lmem.o \
-	$(OBJDIR)/src/host/lua-5.3.0/src/loadlib.o \
-	$(OBJDIR)/src/host/lua-5.3.0/src/lobject.o \
-	$(OBJDIR)/src/host/lua-5.3.0/src/lopcodes.o \
-	$(OBJDIR)/src/host/lua-5.3.0/src/loslib.o \
-	$(OBJDIR)/src/host/lua-5.3.0/src/lparser.o \
-	$(OBJDIR)/src/host/lua-5.3.0/src/lstate.o \
-	$(OBJDIR)/src/host/lua-5.3.0/src/lstring.o \
-	$(OBJDIR)/src/host/lua-5.3.0/src/lstrlib.o \
-	$(OBJDIR)/src/host/lua-5.3.0/src/ltable.o \
-	$(OBJDIR)/src/host/lua-5.3.0/src/ltablib.o \
-	$(OBJDIR)/src/host/lua-5.3.0/src/ltm.o \
-	$(OBJDIR)/src/host/lua-5.3.0/src/lundump.o \
-	$(OBJDIR)/src/host/lua-5.3.0/src/lutf8lib.o \
-	$(OBJDIR)/src/host/lua-5.3.0/src/lvm.o \
-	$(OBJDIR)/src/host/lua-5.3.0/src/lzio.o \
 	$(OBJDIR)/src/host/os_chdir.o \
 	$(OBJDIR)/src/host/os_copyfile.o \
 	$(OBJDIR)/src/host/os_getcwd.o \
